Classes/Cache/Frontend/VariableFrontendWithWarmupReservation.php

```diff
@@ -59,4 +71,14 @@ public function flushByTags(array $tags)
         $queueService = GeneralUtility::makeInstance(QueueService::class);
         $queueService->queueMany($urls);
     }
+    
+    private function areAllGetParamsAllowed($whitelistedGetParams, $getParams): bool
+    {
+        return array_intersect($getParams, $whitelistedGetParams) === $getParams;
+    }
+
+    private function getWhitelistedParams(): string
+    {
+        return $this->extensionConfiguration['getparams']['whitelisterparams'];
```

Comment:
I guess this will result in a fatal error, when the setting is not available. You could catch it with `?? ''`.

Reply:
fixed